exchanges/protocol/features.go

```diff
@@ -3,6 +3,7 @@ package protocol
 // Features holds all variables for the exchanges supported features
 // for a protocol (e.g REST or Websocket)
 type Features struct {
+	// TickerBatching allows the REST endpoint to fetch the entire ticker list available to the exchange
```

Comment:
🐒 
* I don't think this should say REST
* Are we really saying it's *always* the entire thing. That feels like an exchange implementation thing. I think all we're saying here is "the exchange protocol allows batching". I could be wrong.
* Typo at the end of the sentence, but we can omit it.
Guessing we want:
```suggestion
	// TickerBatching allows api to fetch the tickers in batches
```

Though ... I mean ... it's not adding any information for me 🤷

Reply:
The state of this value doesn't seem to be directly used by our engine anywhere, so we don't functionally enshrine a definition that way.

It's hard to tell exactly why I added that comment, as it was written 20 months ago, but I suspect that I didn't understand exactly what that meant, asked shazbert/thrasher, and they responded with that. But really, even now, I wouldn't consider myself to have an authoritative view on what the definition ought to be. Maybe wait for input from others?

Reply:
I think there's a possible confusion; "allows batching" as in "you request a batch of tickers for a pair, and can receive up to 50 ticks for that pair", or as in "you can simultaneously request the latest ticker for every pair on the exchange", or even "you can simultaneously request the latest ticker for up to N pairs on the exchange".

I expect that's what I was initially confused about.

Reply:
In all cases it's "you can request the latest tickers for every pair on the exchange in a single batched request". Can't think of any exchanges which require pagination for doing a bulk request of tickers off the top of my head

Reply:
Ah, I seem to have confused it with candles, my bad.

Yeah it should be fine without a comment then.